kdumpctl

```diff
@@ -1654,6 +1657,65 @@ _should_reset_crashkernel() {
 	[[ $(kdump_get_conf_val auto_reset_crashkernel) != no ]] && systemctl is-enabled kdump &> /dev/null
 }
 
+check_vmcore_creation_status()
+{
+	local _status _timestamp _status_date
+
+	[[ ${VMCORE_CREATION_NOTIFICATION,,} == "yes" ]] || return
+
+	if [[ ! -s $VMCORE_CREATION_STATUS ]]; then
+		dwarn "Notice: No vmcore creation test performed!"
+		return
+	fi
+
+	read -r _status _timestamp < "$VMCORE_CREATION_STATUS"
+	_status_date="$(date -d "@$_timestamp")"
+
+	if [[ "$_status" == "success" ]]; then
+		dinfo "Notice: Last successful vmcore creation on $_status_date"
+	else
+		dwarn "Notice: Last unsuccessful vmcore creation on $_status_date"
+	fi
+}
+
+kdump_test()
+{
+	local _dir
+
+	if ! is_kernel_loaded "$DEFAULT_DUMP_MODE"; then
+		derror "Kdump needs be operational before test."
+		exit 1
+	fi
+
+	_dir=$(dirname "$VMCORE_CREATION_STATUS")
+	if ! [[ -d "$_dir" ]]; then
+		derror "Vmcore status dir $_dir not exist."
+		exit 1
+	fi
+
+	lsblk $(get_mount_info SOURCE target "$_dir") > /dev/null
+
+	if [[ $? -ne 0 ]]; then
```

Comment:
You can check the return value directly with
```
if ! lsblk $(get_mount_info SOURCE target "$_dir") > /dev/null; then
...
```

Reply:
Thanks, much better!